tools/filter_by_fasta_ids/filter_by_fasta_ids.py

```diff
@@ -64,14 +63,21 @@ def main():
     parser.add_argument('-d', help='Path to discarded entries file')
     header_criteria = parser.add_mutually_exclusive_group()
     header_criteria.add_argument('--id_list', help='Path to the ID list file')
+    parser.add_argument('--pattern', default=None, help='regex earch attern for ID in Fasta entry')
```

Comment:
"search pattern" ... mention the implicit default value as well?

Reply:
Modified